bits/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,102 @@
+package sendgrid
+
+import (
+	"context"
+	"net/http"
+
+	"encore.dev/beta/errs"
+	"encore.dev/pubsub"
+	"encore.dev/rlog"
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+// Encore docs about secrets: https://encore.dev/docs/primitives/secrets
+var secrets struct {
+	SendGridAPIKey string
+}
+
+type SendParams struct {
+	From struct {
+		Name    string `json:"name,omitempty"`
+		Address string `json:"address"`
+	} `json:"from"`
+	To struct {
+		Name    string `json:"name,omitempty"`
+		Address string `json:"address"`
+	} `json:"to"`
```

Comment:
To make it easier to call this from other Go services, these should be a named type (`Address`). Maybe make the `Address` field be named `Email` instead to avoid overloading it?

Reply:
Thanks, changed 👍 
5162819bdc7367db5cd47e03f4706a2f4fed5184